jdbc/src/main/java/org/apache/iotdb/jdbc/Config.java

```diff
@@ -52,6 +52,6 @@ private Config(){}
 
   public static final String JDBC_DRIVER_NAME = "org.apache.iotdb.jdbc.IoTDBDriver";
 
-  public static boolean rpcThriftCompressionEnable = false;
+  public static boolean rpcThriftCompressionEnable = true;
```

Comment:
why set it as true?

Reply:
It is said in thrift that when the thrift structures are large and sent frequently, it will be overall better to enable compression, so I made the default compression true.